tests/e2e/util/kubectl/kubectl.go

```diff
@@ -41,8 +49,8 @@ func kubectl(format string, args ...interface{}) string {
 }
 
 // CreateFromString creates a resource from the given yaml string
-func CreateFromString(yamlString string) error {
-	cmd := kubectl("create -f -")
+func CreateFromString(yamlString string, kubeconfig ...string) error {
```

Comment:
@luksa  I opened a new Issue to refactor `kubectl` to manage better the kubeconfig after this PR is merged. I think is big enough to add more changes here